### PR TITLE
ApplicationPagedList-seeders-and-some-chenges

### PR DESCRIPTION
**Pull request by Muhammadislom99**

### CODE REVIEW

File: src/MRA.Jobs.Client/Pages/Admin/Users/UserProfile.razor.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ protected override async Task OnInitializedAsync()
                     return new { Key = parts[0], Value = parts[1] };
                 }).ToDictionary(x => x.Key, x => x.Value.Replace("=", ""));
 
-                _selectedVacancy.Title = filterDictionary.GetValueOrDefault("Vacancy.Title");
+                _selectedVacancy.Title = filterDictionary.GetValueOrDefault("Vacancy.Title").Replace("=", "");
```
Comment (Firuz2006) [inline src/MRA.Jobs.Client/Pages/Admin/Users/UserProfile.razor.cs:62]: What is that?

File: src/MRA.Jobs.Client/Pages/Applicant/Jobs.razor [hunk 2]
```diff
@@ -190,7 +190,8 @@
                     }
                     else
                     {
-                        <MudText Typo="Typo.h5" Style="margin:50px;">@ContentService["VacanciesList:NoJobs"]</MudText>
+                        @* <MudText Typo="Typo.h5" Style="margin:50px;">@ContentService["VacanciesList:NoJobs"]</MudText> *@
```
Comment (Firuz2006) [inline src/MRA.Jobs.Client/Pages/Applicant/Jobs.razor:193]: Comment

File: src/MRA.Jobs.Client/Pages/Applications/Applications.cs [hunk 3]
```diff
@@ -81,7 +82,7 @@ protected override async Task OnInitializedAsync()
                 return new { Key = parts[0], Value = parts[1] };
             }).ToDictionary(x => x.Key, x => x.Value);
 
-            _selectedVacancy.Title = filterDictionary.GetValueOrDefault("Vacancy.Title");
+            _selectedVacancy.Title = filterDictionary.GetValueOrDefault("Vacancy.Title").Replace("=", "");
```
Comment (Firuz2006) [inline src/MRA.Jobs.Client/Pages/Applications/Applications.cs:85]: ?

**Review by Firuz2006 — APPROVED** [review]

File: src/MRA.Jobs.Client/Pages/Applications/Applications.cs (hunk 3 above)
Comment (Muhammadislom99) [reply src/MRA.Jobs.Client/Pages/Applications/Applications.cs:85]: https://www.notion.so/silkroadprofessionals/Filter-in-the-application-list-6e5c44596ccc4ee5bf54bcf289c23848?pvs=4 
image

**Pull request merged**